kafka-rest/src/test/java/io/confluent/kafkarest/integration/KafkaRestContextIntegrationTest.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.kafkarest.integration;
+
+import static java.util.Collections.singletonList;
+import static org.easymock.EasyMock.anyObject;
+import static org.easymock.EasyMock.expect;
+import static org.easymock.EasyMock.mock;
+import static org.easymock.EasyMock.replay;
+import static org.junit.Assert.assertEquals;
+
+import io.confluent.kafkarest.KafkaRestConfig;
+import io.confluent.kafkarest.KafkaRestContext;
+import io.confluent.kafkarest.ProducerPool;
+import io.confluent.kafkarest.backends.kafka.KafkaModule;
+import io.confluent.kafkarest.common.KafkaFutures;
+import io.confluent.kafkarest.extension.RestResourceExtension;
+import io.confluent.kafkarest.v2.KafkaConsumerManager;
+import io.confluent.rest.exceptions.RestException;
+import java.util.Properties;
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerRequestFilter;
+import javax.ws.rs.core.Configurable;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import org.apache.kafka.clients.admin.Admin;
+import org.apache.kafka.clients.admin.DescribeClusterResult;
+import org.apache.kafka.clients.producer.Producer;
+import org.junit.Test;
+
+public class KafkaRestContextIntegrationTest extends ClusterTestHarness {
+
+  /** HTTP 418 I'm a teapot */
+  private static final int I_M_A_TEAPOT_STATUS_CODE = 418;
+
+  private DescribeClusterResult describeClusterResult;
```

Comment:
nit: I think this is redundant, hidden by a local variable.

Reply:
Done.